src/devices/Common/Iot/Device/Common/FlagsHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Common
+{
+    /// <summary>
+    /// Contains functionality to operate on a bit field <see cref="Enum"/>,
+    /// which is an <see cref="Enum"/> decorated with the <see cref="FlagsAttribute"/>
+    /// and which represents a set of flags.
+    /// </summary>
+    public static class FlagsHelper
+    {
+        /// <summary>
+        /// Determines whether one or more bit fields are set in the provided <paramref name="flags"/> bit field.
+        /// </summary>
+        /// <typeparam name="T"><see cref="Enum"/> decorated with the <see cref="FlagsAttribute"/>.</typeparam>
+        /// <param name="flags">Bit field to use as basis for the lookup.</param>
+        /// <param name="flag">One or more bit fields to lookup.</param>
+        public static bool IsSet<T>(T flags, T flag)
```

Comment:
why not use HasFlags directly? It's longer this way and more instructions? (`foo.HasFlags(...)` vs `FlagsHelper.IsSet(foo, ...)`)

Reply:
I've tried to make a consistent mini API to work with flags, therefore the IsSet method.